### PR TITLE
Update Boost for daint 20.11 after update

### PR DESCRIPTION
**Pull request by aurianer**

### CODE REVIEW

**Comment by aurianer** [comment]
My bad seems like 20.11 not in production yet..

File: .jenkins/cscs/env-clang-cuda.sh [hunk 1]
```diff
@@ -9,12 +9,12 @@ source $SPACK_ROOT/share/spack/setup-env.sh
 export CRAYPE_LINK_TYPE=dynamic
 export APPS_ROOT="/apps/daint/SSL/HPX/packages"
 export CXX_STD="17"
-export BOOST_ROOT="${APPS_ROOT}/boost-1.73.0-gcc-8.3.0-c++17-release"
 export HWLOC_ROOT="${APPS_ROOT}/hwloc-2.0.3-gcc-8.3.0"
 
 module load daint-gpu
 module load cudatoolkit/10.2.89_3.29-7.0.2.1_3.5__g67354b4
 module switch cce cce/10.0.2
+module load Boost/1.75.0-CrayGNU-20.11
```
Comment (msimberg) [inline .jenkins/cscs/env-clang-cuda.sh:17]:
```suggestion
module load Boost/1.75.0-CrayCCE20.11
```

Comment (aurianer) [reply .jenkins/cscs/env-clang-cuda.sh:17]: thanks!

**Review by msimberg — APPROVED** [review]
Thanks!

**Pull request merged**